pkg/apiserver/rest/usecase/application.go

```diff
@@ -369,6 +369,9 @@ func (c *applicationUsecaseImpl) CreateApplication(ctx context.Context, req apis
 
 // CreateApplicationTrigger create application trigger
 func (c *applicationUsecaseImpl) CreateApplicationTrigger(ctx context.Context, app *model.Application, req apisv1.CreateApplicationTriggerRequest) (*apisv1.ApplicationTriggerBase, error) {
+	if (req.PayloadType == model.PayloadTypeACR || req.PayloadType == model.PayloadTypeDockerhub) && req.ComponentName == "" {
```

Comment:
Because of the need for extension, it is recommended that a list of supported PayloadType be created. 
It is better to provide API to query supported PayloadType.